### PR TITLE
enhancement/mui-404-page

### PR DESCRIPTION
**Pull request by Lil5166**

### CODE REVIEW

File: src/components/pages/404-page/NotFoundPage.styles.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  width: '353px',
+  height: '62px',
+  fontStyle: 'normal',
+  fontWeight: 400,
+  fontSize: '24px',
+  lineHeight: '128%',
+  textAlign: 'center',
+  marginTop: '6%',
+  marginBottom: '57px',
+};
```
Comment (ria0308) [inline src/components/pages/404-page/NotFoundPage.styles.ts:27]: import text styles from typography instead

File: src/components/pages/404-page/NotFoundPage.tsx [hunk 2]
```diff
@@ -1,38 +1,52 @@
...
+import Button from '@/components/common/ui/button-mui';
+import {
+  ButtonColor,
+  ButtonSize,
+  ButtonVariant,
+} from '@/components/common/ui/button-mui/types';
+
+import * as stylesMui from './NotFoundPage.styles';
 
 import styles from './NotFoundPage.module.scss';
```
Comment (ria0308) [inline src/components/pages/404-page/NotFoundPage.tsx:16]: do we need this?

File: src/components/pages/404-page/NotFoundPage.module.scss [hunk 3]
```diff
@@ -28,7 +28,6 @@
 .frog-image{
   position:relative;
   aspect-ratio: 1.2/1;
-  width:69%;
```
Comment (ria0308) [inline src/components/pages/404-page/NotFoundPage.module.scss:31]: do we need this file at all?

**Review by ria0308 — APPROVED** [review]

**Review by kujo205 — CHANGES_REQUESTED** [review]
image
mobile version is not taken into account, use [breakpoints](https://mui.com/material-ui/customization/breakpoints/) to adjust to users screen size, u may find all the breakpoints in `src/styles/theme/constants/breakpoints/breakpoints.ts` file and example of usage is [here](https://github.com/fictadvisor/fictadvisor-web/blob/d3fd5748af6438c94c95b93c40ee1a82fc006478/src/components/pages/account-page/components/group-tab/components/text-area-popup/TextAreaPopup.styles.ts#L12)

**Review by kujo205 — APPROVED** [review]

File: src/components/pages/404-page/NotFoundPage.styles.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  backgroundRepeat: 'no-repeat, no-repeat, no-repeat',
+  backgroundPosition: 'left bottom, right bottom, center center',
+  backgroundSize: '29%, 29%, 100%',
+};
+
+export const notFoundText: SxProps<Theme> = {
+  width: { desktop: '353px', mobile: '226px' },
+  height: { desktop: '62px', mobile: '54px' },
+  textAlign: 'center',
+  marginTop: { desktop: '64px', mobile: '63px' },
```
Comment (ria0308) [inline src/components/pages/404-page/NotFoundPage.styles.ts:22]: change to 64px in all cases

File: src/components/pages/404-page/NotFoundPage.tsx [hunk 5]
```diff
@@ -1,42 +1,58 @@
...
           <Button
-            text={"Зв'язатися з адміністратором"}
+            text="Зв'язатися з адміністратором"
+            variant={ButtonVariant.FILLED}
             size={ButtonSize.LARGE}
+            sx={{
+              maxHeight: '56px',
+              whiteSpace: 'nowrap',
+              borderRadius: '8px',
+              maxWidth: '300px',
```
Comment (ria0308) [inline src/components/pages/404-page/NotFoundPage.tsx:49]: move to styles.ts

File: src/components/pages/404-page/NotFoundPage.styles.ts [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  marginBottom: { desktop: '57px', mobile: '44px' },
+  typography: { desktop: 'h5', mobile: 'body2' },
```
Comment (Yevheniy2020) [inline src/components/pages/404-page/NotFoundPage.styles.ts:24]: you need replace all typography as 
import theme from '@/styles/theme';
`typography: { desktop: theme.typography.h5, mobile:  ...},`

File: src/components/pages/404-page/NotFoundPage.tsx [hunk 7]
```diff
@@ -1,42 +1,58 @@
...
+            variant={ButtonVariant.FILLED}
             size={ButtonSize.LARGE}
             color={ButtonColor.SECONDARY}
-            startIcon={<ChevronLeftIcon className={'icon'} />}
+            startIcon={<ChevronLeftIcon />}
+            sx={{
+              maxHeight: '56px',
+              whiteSpace: 'nowrap',
+              borderRadius: '8px',
+              maxWidth: '300px',
```
Comment (ria0308) [inline src/components/pages/404-page/NotFoundPage.tsx:36]: move to styles.ts

**Review by Yevheniy2020 — APPROVED** [review]

**Pull request merged**